### DISCUSSION TITLE
NODE_ENV production triples bundle size

### DISCUSSION THREAD

**Discussion by ryands17** [serverless/serverless#9123] [Q&A]
I have created a repo [here](https://github.com/ryands17/serverless-prisma/) that deploys a GraphQL API using Apollo and Prisma. `sls package` creates a bundle of around 45 MB but `NODE_ENV=production sls package` creates a bundle of 128 MB that fails to upload due to Lambda limits. Could anyone help here debug why this is being caused?

Without `NODE_ENV`:

![image](https://user-images.githubusercontent.com/19697099/111316841-9dbd4300-8689-11eb-94e5-5a3b1745f82f.png)

With `NODE_ENV` set to `production`:

![image](https://user-images.githubusercontent.com/19697099/111316976-b9c0e480-8689-11eb-9214-c6e4a2bce4d7.png)

**Comment by pgrzesik** [comment]
Hello @ryands17 :wave: I've reproduced this exact issue and I'm pretty sure the `NODE_ENV=production` alters the output of `npm ls` runs that are used during dev dependencies exclusion - would you be willing to submit an issue on issue tracker so we can take a closer look and verify if it's a bug? It would be much appreciated :bow:

**Reply by ryands17** [reply]
Sure. Will do :)

**Reply by pgrzesik** [reply]
Thanks :raised_hands:

**Reply by ryands17** [reply]
Done: https://github.com/serverless/serverless/issues/9134
